tools/pv_tree_generator/PVtree_test.sh

```diff
@@ -0,0 +1,3 @@
+pip install parameterized
+python3 -m pytest
```

Comment:
I think you need pip install pytest here as well. How about making a requirements.txt and put all the dependency in it.

Reply:
done